themes/app/templates/App/Controller/Layout/CustomTaxonomyDirectoryController.ss

```diff
@@ -0,0 +1,9 @@
+<h1>News Listing Page</h1>
+
+<h2>Results for '$Title'</h2>
```

Comment:
I just realised a quick fix we could add here is to check if `$Title` exists, if so then render otherwise dont display :)